adding-a-new-module/template/src/androidLibAndroidTest/kotlin/TestActivity.kt

```diff
@@ -0,0 +1,34 @@
+package com.splendo.kaluga.template
+
+import android.os.Bundle
+import androidx.appcompat.app.AppCompatActivity
+
+/*
+
+Copyright 2019 Splendo Consulting B.V. The Netherlands
+
+   Licensed under the Apache License, Version 2.0 (the "License");
+   you may not use this file except in compliance with the License.
+   You may obtain a copy of the License at
+
+     http://www.apache.org/licenses/LICENSE-2.0
+
+   Unless required by applicable law or agreed to in writing, software
+   distributed under the License is distributed on an "AS IS" BASIS,
+   WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+   See the License for the specific language governing permissions and
+   limitations under the License.
+
+*/
+
+class TestActivity : AppCompatActivity() {
+    // val viewModel: <Concrete ViewModel here> by viewModels()
+
+    override fun onCreate(savedInstanceState: Bundle?) {
+        super.onCreate(savedInstanceState)
+    }
+
+    override fun onDestroy() {
+        super.onDestroy()
+    }
```

Comment:
Not sure what the benefit of pre declaring these are (espc onDestroy)

Reply:
I'll remove onDestroy. onCreate is probably needed in 99.99(9)% of cases.

Reply:
Thinking about this more, you often don't need an Activity at all (at least `location`, `logging`, `test-utils` don't, and two do have tests here).  Should this even be in here?

Reply:
Yep, sometimes we don't need instrumented tests, but I would prefer to live it in a template. Delete is allways easier than add

Reply:
Template for instrumented tests is fine (to have the directories etc) but specifically having an Activity goes a bit far. It's just a plain Activity.

It's also missing from the manifest incidently.

Reply:
onRemove is now gone, but now there's just an empty activity (not in any manifest still right?)

Reply:
should be in /androidLibAndroidTest/AndroidManifest.xml